File: app/models/timezone.rb
class Timezone < ApplicationRecord
  belongs_to :user

  validates :name, presence: true
  validates :city, presence: true
  validates :difference, numericality: {
      greater_than_or_equal_to: -12,
      less_than_or_equal_to: 12,
      only_integer: true
  }
end

File: client/app/models/timezone.js
import Model from 'ember-data/model';
import attr from 'ember-data/attr';
import { belongsTo } from 'ember-data/relationships';

export default Model.extend({
  name: attr('string'),
  city: attr('string'),
  difference: attr('number'),
  user: belongsTo('user')
});
